dev/com.ibm.ws.concurrent.persistent_fat_multiple/fat/src/com/ibm/ws/concurrent/persistent/fat/multiple/MultiplePersistentExecutorsTest.java

```diff
@@ -33,23 +36,72 @@
  * Tests for multiple persistent scheduled executor instances sharing the same database
  */
 @RunWith(FATRunner.class)
-@SkipIfSysProp(DB_Informix) // persistent executor is not support on Informix
+@SkipIfSysProp({DB_Informix, DB_Sybase, "fat.oracle.is.available=false"})
```

Comment:
checking `fat.oracle.is.available=false` will work when running locally, but not for remote builds because we build the FAT on one machine and then run it potentially multiple times on a different machine at a later time